server/portal/apps/googledrive_integration/views.py

```diff
@@ -57,17 +58,19 @@ def initialize_token(request):
 @csrf_exempt
 @login_required
 def oauth2_callback(request):
+    error_message = 'An unexpected error occurred during Google Drive setup.'
```

Comment:
The message defined by Design was different ([source](https://jira.tacc.utexas.edu/browse/FP-952#changehistory-144776)):
> An unexpected error occurred while trying to set up Google Drive. Please try again.

I've asked Design whether they have concern with the change to the first sentence.

**Is it likely futile for the user to try again?**

Reply:
2. **Can TACC technical support assist the user (if trying again is not helpful)?**
   (If so, Design will offer some kind of "please contact support" statement.)